tailwind.config.js

```diff
@@ -200,10 +200,15 @@ module.exports = {
           '50%': { opacity: '0' },
           '100%': { opacity: '1' },
         },
+        wiggle: {
```

Comment:
Where is this used

Reply:
Was used in the previous animation, removed this block now.